### ISSUE TITLE
your master branch also has dependency problem.

### ISSUE THREAD

**Issue by Masanori-Yoshihira** [ethnhe/PVN3D#70]
As I pointed out to you yesterday, your torch-1.5 has bugs and missing files.

[TypeError: step() missing 1 required positional argument: 'step'#69](https://github.com/ethnhe/PVN3D/issues/69)

Everybody who find your program may expect it to work. However, even if they follow your INSTALL procedure, no one will be able to get your program to work properly.

The readme says that pointnet2+ works with pytorch 1.0, but the master branch of pointnet2+ only works with pytorch 1.4 or higher. v1.0 of pointnet2+ does not work with pytorch 1.0 due to a change in the use of the torch.utils.ffi file. v1.0 of pointnet2+ says that it works with pytorch 0.4, but it does not work with pytorch 0.4 will not work with python3.

In conclusion, this PVN3D will not work with either the master branch or the pytorch-1.5 branch, and everyone who wants to use it should be well aware that it will not work at all due to bugs in the program, missing files, and impossible version combinations. If somebody want to use this program, they should be aware that all branch will not work at all due to bugs in the program, missing files, or impossible version combinations.

**Comment by Masanori-Yoshihira** [comment]
please tell me how to install pointnet2+ for your pvn3d master. as i wrote above, there is any mismatch in version of pytorch. 

-question1:
which version did you install for pvn3d? 

-question2:
which branch did you install on pointnet2+?

-question3:
how did you install for question1 and question2?

I am not beginner for programming. your readme doesn't show correct procedure. i need your help.

**Comment by ethnhe** [comment]
Your conclusion is too subjective and the tongue is mean and aggressive, not as a person asking for help. I'll answer the questions though.

The question in issue is replied in #69

> Everybody who find your program may expect it to work. However, even if they follow your INSTALL procedure, no one will be able to get your program to work properly.

Did you really follow the INSTALL procedure in our README? If you follow it and simply run ```python3 setup.py build_ext``` in the folder ```PVN3D```, the work is done! And if you read the simple code in ```PVN3D/setup.py``` you will find we keep the source code of PointNet++ for a stable version in pytorch1.0 in ```pvn3d/_ext-src```, which is from this [commit](https://github.com/erikwijmans/Pointnet2_PyTorch/tree/88406aa4390710a54ffafd30c19662074ceee0cb).
